lib/t/struct/acts_as_comparable.rb

```diff
@@ -7,24 +7,16 @@ module ActsAsComparable
       extend T::Sig
       include ::Comparable
 
-      LESS_THAN_OTHER = -1
-      EQUAL = 0
+      NOT_COMPARABLE = nil
 
       sig { params(other: Object).returns(Integer) }
```

Comment:
Noting that this will need updating to be:

```suggestion
      sig { params(other: Object).returns(T.nilable(Integer)) }
```